docs/api/printer.rst

```diff
@@ -362,6 +362,8 @@ Issue a tool command
      Extrudes the given amount of filament from the currently selected tool. Additional parameters:
 
      * ``amount``: The amount of filament to extrude in mm. May be negative to retract.
+     * ``speed``: Optional. Speed at which to extrude. If not provided, maximum speed for E axis from printer
```

Comment:
Small error - it's mm/min, not mm/s, at least if you directly hand it over to the `F` parameter, which is [defined as mm/min](https://reprap.org/wiki/G-code#G0_.26_G1:_Move). I'm going to just quickly fix that myself though :)

Reply:
Good catch. Should we fix it also in _printer.rst_ in the _Issue a print head command_ for the _jog_ operation? It seems to also use G1 that operates with mm/mm. Let me know if you want me to do a PR for this extra-small change.

Thanks,
Gaston

Reply:
Yes, please do!